readme.md

```diff
@@ -54,7 +66,12 @@ console.log(results);
 
 Your Answer:
 ```text
+In mongoose, interacting with the database requires beginning with "db." to tell mongoose where to look. So, to get this code to work, we could fix it to say:
+```
 
+```js
+var results = db.AuthorModel.find({name: "Bob"});
+console.log(results);
 ```
```

Comment:
Issue here is that `find` is an asynchronous method. `results` is undefined when we try to print it to the console because `find` is not done running yet. We can fix this by passing a callback into `find` like so...

```js
var results = AuthorModel.find({name: "Bob"}, function(results){
  console.log(results);
});
```